colors/oak.vim

```diff
@@ -129,6 +129,12 @@ call s:hi('LspDiagnosticsUnderlineError',       s:petal1, '', 'underline')
 call s:hi('LspDiagnosticsUnderlineWarning',     s:petal2, '', 'underline')
 call s:hi('LspDiagnosticsUnderlineInformation', s:leaf2, '', 'underline')
 call s:hi('LspDiagnosticsUnderlineHint',        s:fur4, '', 'underline')
+if g:oak_virtualtext_bg == 1
+  call s:hi('LspDiagnosticsVirtualTextError',       s:petal1, '#2B190A')
+  call s:hi('LspDiagnosticsVirtualTextWarning',     s:petal2, '#2B190A')
+  call s:hi('LspDiagnosticsVirtualTextInformation', s:leaf2,  '#2B190A')
+  call s:hi('LspDiagnosticsVirtualTextHint',        s:fur4,   '#2B190A')
```

Comment:
Why can't we use `trunk2` here ?

Reply:
too bright

Reply:
lemme show you

Reply:
![Screenshot from 2020-11-24 18-48-12](https://user-images.githubusercontent.com/36493671/100099576-a6450000-2e85-11eb-9224-870601c804b0.png)

Reply:
it might not look too bright in the browser because the surrounding webpage is white. check in terminal

Reply:
i'll do that if you want

Reply:
Indeed, too bright...

Reply:
How about using `#2B1C0E` and using that ad `trunk2` ?

Reply:
yeah

Reply:
but this makes virtualtext invisible.

Reply:
let's drop this, people who want it will highlight in their configs